tensorbay/apps/sextant.py

```diff
@@ -111,6 +113,20 @@ def __init__(self, access_key: str, url: str = "") -> None:
         super().__init__(access_key, url)
         self._open_api = urljoin(self.gateway_url, "apps-sextant/v1")
 
+    def _generate_benmarks(
+        self,
+        offset: int = 0,
+        limit: int = 128,
```

Comment:
remove the trailing `,`, `black` will format this function signature into oneline